fission-web-api/library/Fission/Web/API/Auth/Types.hs

```diff
@@ -1,19 +1,25 @@
+{-# OPTIONS_GHC -fno-warn-orphans #-}
```

Comment:
Where "orphan instance" here is because we're defining type aliases. Yes, you could _potentially_ have a conflicting instance elsewhere, but you shouldn't use it! Such a compiler error would be correct. We could create a custom type for these, but then that won't plug into the auth subsystem. I'm sure it's possible with enough effort, but this feels fine given the context.